src/network/networkd-netdev-bridge.c

```diff
@@ -72,20 +72,21 @@ static int netdev_bridge_post_create(NetDev *netdev, Link *link, sd_netlink_mess
         if (r < 0)
                 return log_netdev_error_errno(netdev, r, "Could not append IFLA_INFO_DATA attribute: %m");
 
+        /* convert to jiffes */
         if (b->forward_delay > 0) {
-                r = sd_netlink_message_append_u32(req, IFLA_BR_FORWARD_DELAY, b->forward_delay / USEC_PER_SEC);
+                r = sd_netlink_message_append_u32(req, IFLA_BR_FORWARD_DELAY, b->forward_delay / (sysconf(_SC_CLK_TCK) * 100));
```

Comment:
I really think we shouldn't use `sysconf()` every time we do that division.  After all, we don't expect that value to change during runtime, do we?

Instead, jiffies/second or something similar should be determined once in a convenience function in `time-util.c`, and then stored in a `static` variable, so it is cached for subsequent calls.